environments/icds-new/proxy.yml

```diff
@@ -2,7 +2,7 @@ SITE_HOST: 'www.icds-cas.gov.in'
 NO_WWW_SITE_HOST: 'icds-cas.gov.in'
 primary_ssl_env: "cas"
 
-nginx_max_clients: 1024
+nginx_max_worker_connection: 2048
```

Comment:
I think you meant to make this change in `icds-cas` not `icds-new`

Reply:
I got confused with `new` :p